README.md

```diff
@@ -6,76 +6,215 @@
+
+## Dependabot CLI
+
+The [Dependabot CLI][dependabot-cli] is a newer tool that may eventually replace `dependabot-script` for standalone use cases.
+While it creates dependency diffs, it's currently missing the logic to turn those diffs into actual PR's. Nevertheless, it
+may be useful for advanced users looking for examples of how to hack on Dependabot.
+
+# Contributing to Dependabot
+
+## Reporting issues and Feature Requests
+
+👋 Want to give us feedback on Dependabot, or contribute to it? That's great - thank you so much!
+
+### Reproducible Example
 
-### Disclosing security issues
+Most bug reports should be accompanied by a link to a public repository that reproduces the problem. Bug reports that
+cannot be reproduced on a public repo using the [dry-run script](TODO:hyperlink) may be closed as "cannot reproduce".
+
+TODO: is there anything else we specifically want here? Generally as long as it's public it's been quick for us to repro,
+as the dry-run script allows specifying exactly which libs to try to bump, so we can specify the failing one to zero in on
+it quickly w/o needing them to manually simplify their manifest...
+
+### No Plus One
+
+Our issue tracker is quite active, and as a result there's a good chance someone already filed the same issue. If so,
+please upvote that issue, because we use 👍 reactions on issues as one signal to gauge the impact of a feature request or bug.
+
+However, please do not leave comments that contribute nothing new to the discussion. For details, see https://github.com/golang/go/wiki/NoPlusOne. This is open source, if you see something you want fixed, we are happy to coach you through contributing a pull request to fix it.
+
+### Don't file issues about Security Alerts or Dependency Graph
+
+The issue-tracker is meant solely for issues related to Dependabot's updating logic. Issues about security alerts or Dependency Graph should instead be filed as a [Code Security discussion](https://github.com/orgs/community/discussions/categories/code-security).
+
+A good rule of thumb is that if you have questions about the _diff_ in a PR, it belongs here.
+
+### Disclosing Security Issues
 
 If you believe you have found a security vulnerability in Dependabot please submit the vulnerability to GitHub Security [Bug Bounty](https://bounty.github.com/) so that we can resolve the issue before it is disclosed publicly.
 
-## What's in this repo?
+## Submitting Pull Requests
+
+Want to contribute to Dependabot? That's great - thank you so much!
+
+Contribution workflow:
+1. Fork the project.
+2. Get the [development environment running](TODO:hyperlink).
+3. Make your feature addition or bug fix.
+4. Add [tests for it](TODO:hyperlink). This is important so we don't break it in a future version unintentionally.
+5. Send a pull request. The tests will run on it automatically, so don't worry if you couldn't get them running locally.
```

Comment:
This is duplicated here and in the Contributing section... Kinda like to drop one of them, but not sure which one... I do like having a contribution workflow front and center on the readme.